core/proto/sequencer_v1_grpc/sequencer_v1_grpc.proto

```diff
@@ -14,9 +14,11 @@
 
 syntax = "proto3";
 
+option go_package = "github.com/google/keytransparency/core/proto/sequencer_v1_grpc";
+
 package sequencer.v1.grpc;
 
-import "github.com/google/keytransparency/core/proto/keytransparency_v1_proto/keytransparency_v1_proto.proto";
+import "proto/keytransparency_v1_proto/keytransparency_v1_proto.proto";
```

Comment:
sort order (should be below google/api/...)

Reply:
done